src/components/Main.jsx

```diff
@@ -43,6 +43,32 @@ class MainComponent extends Component {
     }
   }
 
+  printTemplate=()=>{
+    var prtContent = document.getElementById("template");
+    var WinPrint = window.open('', '', 'left=0,top=0,width=700,height=900, toolbar=0,scrollbars=0,status=0');
+    WinPrint.document.write(`<html style='-webkit-print-color-adjust: exact;'><head>
+    <link href='https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css' rel='stylesheet'>
+    </head>
+    <style>
+    @page{
+      margin:0;
+    }
+    @media print {
+      html, body {
+        width: 210mm;
+      }
+    }
+    </style>
+    <body>`+prtContent.innerHTML+`</body></html>`);
+
+    WinPrint.document.close();
+    WinPrint.focus();
+    setTimeout(()=>{
+      WinPrint.print();
+      WinPrint.close();
+    },1000)
+  }
```

Comment:
Cannot able to understand what this is doing. Please extract this into separate method. You can create a new class in a separate file which will take html as argument and it will print the document.